scripts/etc/init.d/rcS

```diff
@@ -85,6 +85,28 @@ sanitize_inputfile()
     fi
 }
 
+install_files()
+{
+  file_to_read="$1"
+  echo "Adding files & folders listed in /bootfs/config/$file_to_read"
+  sanitize_inputfile "/bootfs/config/$file_to_read"
+  grep -v "^[[:space:]]*#\|^[[:space:]]*$" "/bootfs/config/${file_to_read}" | while read -r line; do
+    owner=$(echo "$line" | awk '{ print $1 }')
+    perms=$(echo "$line" | awk '{ print $2 }')
+    file=$(echo  "$line" | awk '{ print $3 }')
+    echo "$file"
```

Comment:
could you add two or four spaces in front of `$file'? that way the list will be indented in the log file and easier to scan through